dev/alias.h

```diff
@@ -1,29 +1,54 @@
 #pragma once
 
-#include <type_traits>  //  std::enable_if, std::is_base_of, std::is_member_pointer
-#include <sstream>  //  std::stringstream
+#include <type_traits>  //  std::enable_if, std::is_base_of, std::is_member_pointer, std::remove_const
+#include <utility>  //  std::index_sequence, std::make_index_sequence
 #include <string>  //  std::string
+#include <sstream>  //  std::stringstream
+#include <algorithm>  //  std::copy_n
 
-namespace sqlite_orm {
+#include "functional/cxx_universal.h"  //  ::size_t
+#include "functional/cxx_type_traits_polyfill.h"
+#include "type_traits.h"
+#include "alias_traits.h"
 
-    /**
-     *  This is base class for every class which is used as a custom table alias, column alias or expression alias.
-     *  For more information please look through self_join.cpp example
-     */
-    struct alias_tag {};
+namespace sqlite_orm {
 
     namespace internal {
 
+#ifdef SQLITE_ORM_WITH_CPP20_ALIASES
```

Comment:
Basically I would say yes.
The problem we will face in the future is the dev to master merge again: since the dev to master merge has already been recorded, the things we removed in the 1.8.2 branch will not be restored and may be lost unless we specifically track the removed features...

Reply:
ok I see. So aliases are going to go to 1.8.2?

Reply:
Like always I found a way:
1. Remove feature from 1.8.2-to-master -> commit X
2. Merge 1.8.2-to-master into dev
3. Revert commit X in dev

Reply:
IMHO it is better to remove feature from `1.8.2-to-master` and then merge it to `master` leaving `dev` with all features we have. Then we make 1.8.2 release, then merge `master` to `dev` to make `dev` contain all commits that `master` have. In that case we will never have `dev` missing any feature and we will not ship aliases to 1.8.2

Reply:
You are right, it's even better to merge `master` into `dev`.
Still, reverting 'commit X' must be done in `dev`.

Shall I remove the whole feature, or simply not define `SQLITE_ORM_WITH_CPP20_ALIASES`?

Reply:
I am not sure how reverting commit works in different branch. I know how it works in a single branch if we want to revert the previous commit: GitHub creates one more commit with reverse diff. So I am
not sure whether commit reversion will work as we expect. I would hope on my hands instead: better remove the feature manually and then add it again. Either way is ok

Reply:
I tried the different work flows. Reverting the commit, then merging `dev` into `master` works fine (so it's like manually adding the feature).

Reply:
ok let's make it this way